src/etl/ETLState.hpp

```diff
@@ -44,7 +44,7 @@ struct ETLState {
      */
     template <typename Forward>
     static std::optional<ETLState>
-    fetchETLStateFromSource(Forward const& source) noexcept
+    fetchETLStateFromSource(Forward& source) noexcept
```

Comment:
can const add back?

Reply:
We can't here because `source` here is either `Source` or `LoadBalancer` and for `LoadBalancer` `forwardToRippled()` is not const.